examples/cdk2-example/cdk2_sams.yaml

```diff
@@ -6,8 +6,8 @@ ligand_file: CDK2_ligands.sdf
 
 #The ligand file contains multiple ligands. Choose the indices of the ligands
 #between which we should compute a relative free energy
-old_ligand_index: 14
-new_ligand_index: 15
+old_ligand_index: LIGANDA_INDEX
```

Comment:
This could be a bit problematic because the job can't be run _without_ using `sed` to replace these---so the YAML file itself isn't valid. A user may want to just run perses directly on this example without having to go through the batch script that runs all pairs.

A better approach might be to include a short python script that does this as needed:
```python
# first, calculate ligand1, ligand2, trajectory_directory, and output_yaml_filename from LSB_JOBINDEX argument

# Read the YAML file, modify it, and write new file
from yaml import load, dump
options = yaml.load('cdk2_sams.yaml')
options['old_ligand_index'] = ligand1
options['new_ligand_index'] = ligand2
options['trajectory_directory'] = trajectory_directory
with open(output_yaml_filename, 'w') as outfile:
    yaml.dump(options, outfile)
```

Reply:
Sure